Microsoft/WebTemplateStudio
# Moved navigation bar to top + incorporated top nav bar design + azure profile link

Changes:
- moved navigation bar to top and made it fully responsive + functional 
- added additional styling to make the transition smoother 
- moved and rerouted the azure profile / login button to cloud services page (will adjust design more in future PR according to Christina's designs)
- changed some text

How to test
- check if the azure login button functions normally on cloud services page 
- UI should look like this: 
![finishedHeader](https://user-images.githubusercontent.com/31115095/60711431-cb6c5380-9ec9-11e9-8f2a-44b6e805f492.PNG)
